src/components/MAPI/clusters/ClusterDetail/ClusterDetailOverview.tsx

```diff
@@ -103,6 +113,35 @@ const ClusterDetailOverview: React.FC<{}> = () => {
         basis='100%'
       />
       <ClusterDetailWidgetCreated cluster={cluster} basis='100%' />
+
+      {cluster && (
+        <Box
+          margin={{ top: 'large' }}
+          direction='column'
+          gap='small'
+          basis='100%'
+          animation={{ type: 'fadeIn', duration: 300 }}
+        >
+          <InspectClusterGuide
+            provider={provider}
+            clusterName={cluster.metadata.name}
+            clusterNamespace={cluster.metadata.namespace!}
+          />
+
+          {releaseVersion && (
```

Comment:
It's a very slim chance that the release version label is not present on the cluster, but we don't take any chances. Just don't render this guide if the release version is not set